packages/rancher-monitoring/rancher-windows-exporter/charts/templates/daemonset.yaml

```diff
@@ -13,65 +12,50 @@ spec:
     metadata:
       labels: {{ include "windowsExporter.labels" . | nindent 8 }}
     spec:
+      hostNetwork: true
       nodeSelector: {{ include "windowsExporter.client.nodeSelector" . | nindent 8 }}
       tolerations: {{ include "windowsExporter.client.tolerations" . | nindent 8 }}
       serviceAccountName: {{ template "windowsExporter.name" . }}
+      initContainers:
+        - name: ensure-port-9796-is-free
+          securityContext: {{ toYaml $.Values.securityContext | nindent 12 }}
+          image: {{ template "system_default_registry" . }}mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0
+          command:
+            - C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe
+          args: ["-f", "scripts/ensure-9796-is-free.ps1"]
+          volumeMounts:
+            - name: exporter-scripts
+              mountPath: /scripts
+        - name: configure-firewall-rule
+          securityContext: {{ toYaml $.Values.securityContext | nindent 12 }}
+          image: {{ template "system_default_registry" . }}mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0
+          command:
+            - C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe
+          args: ["-f", "scripts/configure-firewall.ps1"]
+          volumeMounts:
+            -  name: exporter-scripts
+               mountPath: /scripts
```

Comment:
Note: mount paths and command arguments do not use absolute paths as hosted providers may not mount host process containers in `C:\`. For example, aks uses `C:\C`